modules/order/src/Entity/OrderItemInterface.php

```diff
@@ -99,14 +99,22 @@ public function getUnitPrice();
   public function setUnitPrice(Price $unit_price);
 
   /**
-   * Gets the order item total price.
+   * Gets the order item total price, before adjustments.
    *
    * @return \Drupal\commerce_price\Price|null
    *   The order item total price, or NULL.
    */
   public function getTotalPrice();
 
   /**
+   * Gets the order item total price, with all order item adjustments applied.
+   *
+   * @return @return \Drupal\commerce_price\Price|null
+   *   The order item adjusted total price, or NULL.
+   */
+  public function getTotalAdjustedPrice();
```

Comment:
getAdjustedTotalPrice() sounds better?

No need to change the docblock for total price, that's still the total that people will use.
Instead, we need to explain the adjusted total price, saying that it's for informative use only.
Let's bikeshed it on IRC.